### PR TITLE
Create issue_template.md

### PR DESCRIPTION
**Pull request by ishigamii**
Template for issue always helpful to get more informations from the start

### CODE REVIEW

**Comment by ishigamii** [comment]
@Guichaguri you should merge this in the master btw 👍

**Comment by ishigamii** [comment]
@dcvz An issue template ? :)

**Comment by ishigamii** [comment]
Alo ?

**Comment by Guichaguri** [comment]
Sorry for forgetting about this PR and not answering at all.

I don't think this would help that much. In other projects that have an issue template, most of the people don't fill it properly or don't fill it at all. Some of them become even harder to read through.

Look into [Atom's issue tracker](https://github.com/atom/atom/issues) as an example, a lot of issues have some kind of bad formatting, parts of the template they removed instead of filling in or instructions they forgot to remove. It adds a bureaucracy to opening issues that we don't need yet.

I think it would be better an empty template, with short instructions inside comments (`<!-- -->`), or just a [CONTRIBUTING](https://help.github.com/en/articles/setting-guidelines-for-repository-contributors) file with basic instructions.

I feel like this would be enough for bug reports:
```markdown
<!--
  Please, describe with details the problem, let us know which platform (Android, iOS or Windows) and which OS version are you experiencing this issue.
  Crash logs, step-by-step instructions, code snippets and device details may also help us identify and reproduce the issue.
-->



```

Let me know what do you think.

**Comment by ishigamii** [comment]
I think both are possible issue template, mine doesn't looks so complex cause some are asking so many things that it's impossible to send them a report of a bug cause it's asking lors of work and you give up.
Maybe a mix of both : 
```
# Configuration

Run `react-native info` in your project and share the content.
What `react-native-track-player` version are you using?

# Issue
<!--
  Please, describe with details the problem, let us know which platform (Android, iOS or Windows) and which OS version are you experiencing this issue.
  Crash logs, step-by-step instructions, code snippets and device details may also help us identify and reproduce the issue.
-->

# Code

Show us the code you are using
```

For me though I don't know if it is useful for you the configuration part is very needed, like which version of react-native / your lib the issue has been made with. And it's easy to just launch the command given. We can even remove the code part but it seems useful to me to be able to check if the user didn't forget something ;)

**Comment by dcvz** [comment]
I actually like this new version of the template. @guichari we can try it and see if it works out or helps us.

**Comment by ishigamii** [comment]
updated it

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Configuration
+
+Run `react-native info` in your project and share the content.
+What `react-native-track-player` version are you using?
+
+# Issue
+/*
+  Please, describe with details the problem, let us know which platform (Android, iOS or Windows) and which OS version are you experiencing this issue.
+  Crash logs, step-by-step instructions, code snippets and device details may also help us identify and reproduce the issue.
+*/
```
Comment (Guichaguri) [inline issue_template.md]: Put it between `<!-- -->` instead, since GitHub recognizes it as an actual comment

Comment (ishigamii) [reply issue_template.md]: done (I thought it wasn't visible but it is indeed)

**Comment by Guichaguri** [comment]
Thank you!

**Pull request merged**